config/longrun_configs/cmip_bucket.yml

```
@@ -0,0 +1,20 @@
+FLOAT_TYPE: "Float32"
+aerosol_radiation: true
+atmos_config_file: "config/longrun_configs/longrun_aquaplanet_allsky_0M.yml"
+checkpoint_dt: "720hours"
```

```suggestion
atmos_config_file: "config/longrun_configs/longrun_aquaplanet_allsky_0M.yml"
bucket_albedo_type: "map_temporal"
checkpoint_dt: "720hours"
```
we usually use temporal map albedo for longruns